File: codes/switch-case.c
#include <stdio.h>

int main() {
    char operator = '+';
    int operand1 = 10;
    int operand2 = 5;
    int result;

    switch (operator) {
        case '+':
            result = operand1 + operand2;
            printf("La suma es: %d\n", result);
            break;
        case '-':
            result = operand1 - operand2;
            printf("La resta es: %d\n", result);
            break;
        case '*':
            result = operand1 * operand2;
            printf("La multiplicación es: %d\n", result);
            break;
        case '/':
            result = operand1 / operand2;
            printf("La división es: %d\n", result);
            break;
        default:
            printf("Operador no válido.\n");
            break;
    }

    return 0;
}

File: output/switch-case.py
def main():
	operator = '+'
	operand1 = 10
	operand2 = 5
	
	match operator:
		case '+':
			result = operand1+operand2
			print("La suma es: %d\n" % (result))
		
		case '-':
			result = operand1-operand2
			print("La resta es: %d\n" % (result))
		
		case '*':
			result = operand1*operand2
			print("La multiplicación es: %d\n" % (result))
		
		case '/':
			result = operand1/operand2
			print("La división es: %d\n" % (result))
		
		case _ :
	 		print("Operador no válido.\n")
		
		
	return 0
	
if __name__ == '__main__':
	main()
